### PR TITLE
New versions Fair[MQ|Logger]

### PR DESCRIPTION
**Pull request by dennisklein**

### CODE REVIEW

**Comment by rbx** [comment]
1.4.17 has removed nanomsg transport, so it no longer depends on it.

File: repos/fairsoft/packages/fairmq/package.py [hunk 1]
```diff
@@ -52,7 +61,9 @@ class Fairmq(CMakePackage):
     depends_on('boost@1.64: +container+program_options+filesystem+date_time+regex', when='@1.4:')
     conflicts('^boost@1.70:', when='^cmake@:3.14')
     depends_on('fairlogger@1.2:1.5', when='@:1.4.7')
-    depends_on('fairlogger@1.2:', when='@1.4.8:')
+    depends_on('fairlogger@1.2:', when='@1.4.8:@1.4.10')
```
[the remark this answers is not in the document]
Comment (dennisklein) [reply repos/fairsoft/packages/fairmq/package.py]: fixed

**Comment by dennisklein** [comment]
> 1.4.17 has removed nanomsg transport, so it no longer depends on it.

Thx for the reminder, have modelled it.

**Comment by dennisklein** [comment]
> FairMQ 1.4.19 has a bundled picosha2: Should we try to create a dedicated external package for it?

This would require additional package-level patches. From FairMQ side picosha2 is a private, bundled-only dependency.

**Comment by ChristianTackeGSI** [comment]
> > FairMQ 1.4.19 has a bundled picosha2: Should we try to create a dedicated external package for it?
> 
> This would require additional package-level patches. From FairMQ pov picosha2 is a private, bundle-only dependency.

The question was multi folded:
* Would it make sense to package this (in spack) generally? Not necessarily to be used in FairMQ
* It looks like this is a header only thing. Otherwise, I would have asked, if there is a (shared) library, that should be shared with possible other consumers to avoid ABI conflicts.

**Comment by dennisklein** [comment]
> I would prefer to wait for the tests to complete before merging.

Yep, it is also expected to see potential effects from the PRs #284, #287, and #288 which I have merged recently without waiting for green tests because of the proxy issue in our CI infrastructure.

**Comment by dennisklein** [comment]
hm, root still concretizes with `^openblas@0.3.8`.

**Comment by ChristianTackeGSI** [comment]
> hm, root still concretizes with `^openblas@0.3.8`.

Thanks for noting this!
Same story as with intel-tbb and tbb.
See #292.

**Pull request merged**